### ISSUE TITLE
Since v3.2.0, audio in Windows client isn't working

### ISSUE THREAD

**Issue by iTaybb** [moonlight-stream/moonlight-qt#734]
**Describe the bug**
Since v3.2.0, audio isn't working (isn't being forwarded from server to client, or at least not being played in client).
It would seem that in the first 100ms or so, while still establishing the connection (video not shown yet), audio is being played correctly. After this, it seems like it gets broken/muted.

In v3.1.4 and before this worked fine.

**Affected games**
Everything.

**Client PC details (please complete the following information)**
 - OS: Windows 11 21H2 (also tested yesterday on Windows 10 21H2 and the error persisted there as well)
 - Moonlight Version: v3.2.0
 - GPU: Intel(R) Iris(R) Plus Graphics 655
 - Antivirus and firewall software: Windows Defender. Windows Firewall is disabled.

**Server PC details (please complete the following information)**
 - OS: Windows 10 Pro 21H2
 - GeForce Experience version: 3.25.0.84
 - Nvidia GPU driver: 511.65
 - Antivirus and firewall software: Windows Defender. Windows Firewall is disabled.

**Moonlight Logs (please attach)**
```
00:00:00 - SDL Info (0): Compiled with SDL 2.0.21
00:00:00 - SDL Info (0): Running with SDL 2.0.21
00:00:00 - Qt Info: No translation available for  "he_IL"
00:00:00 - Qt Warning: qrc:/gui/main.qml:12:1: QML ApplicationWindow: ToolTip must be attached to an Item
00:00:00 - Qt Info: Found "gamecontrollerdb.txt" at "C:/Users/user/Downloads/MoonlightPortable-x64-r1685/cache/gamecontrollerdb.txt"
00:00:00 - SDL Info (0): Loaded 430 new gamepad mappings
00:00:00 - SDL Info (0): V-sync disabled
00:00:01 - Qt Info: Found "d3d11_vertex.fxc" at ":/data/d3d11_vertex.fxc"
00:00:01 - Qt Info: Found "d3d11_overlay_pixel.fxc" at ":/data/d3d11_overlay_pixel.fxc"
00:00:01 - Qt Info: Found "d3d11_video_pixel.fxc" at ":/data/d3d11_video_pixel.fxc"
00:00:01 - SDL Info (0): Using D3D11VA accelerated renderer
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 32(VPS), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 33(SPS), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 34(PPS), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 21(CRA_NUT), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 21(CRA_NUT), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 21(CRA_NUT), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] nal_unit_type: 21(CRA_NUT), nuh_layer_id: 0, temporal_id: 0
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Decoding VPS
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Main 10 profile bitstream
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Decoding SPS
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Main 10 profile bitstream
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Decoding VUI
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Decoding PPS
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Format d3d11 chosen by get_format().
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Format d3d11 requires hwaccel initialisation.
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Decoder GUIDs reported as supported:
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {ee27417f-5e28-4e65-beea-1d26b508adc9}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {bf22ad00-03ea-4690-8077-473346209b7e}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {1b81bea4-a0c7-11d3-b984-00c04f2e73c5}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {e07ec519-e651-4cd6-ac84-1370cceec851}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {bcc5db6d-a2b6-4af0-ace4-adb1f787bc89}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {1b81be94-a0c7-11d3-b984-00c04f2e73c5}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {1b81bea2-a0c7-11d3-b984-00c04f2e73c5}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {1b81be68-a0c7-11d3-b984-00c04f2e73c5}00:00:01 - FFmpeg:  10000:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {d79be8da-0cf1-4c81-b82a-69a4e236f43d}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {f9aaccbb-c2b6-4cfc-8779-5707b1760552}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {705b9d82-76cf-49d6-b7e6-ac8872db013c}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {c528916c-c0af-4645-8cb2-372b6d4adc2a}00:00:01 - FFmpeg:  10000:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {91cd2d6e-897b-4fa1-b0d7-51dc88010e0a}00:00:01 - FFmpeg:  10000:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg:  10700:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {90b899ea-3a62-4705-88b3-8df04b2744e7}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {442b942a-b4d9-4940-bc45-a882e5f919f3}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {97688186-56a8-4094-b543-fc9daaa49f4b}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {1424d4dc-7cf5-4bb1-9cd7-b63717a72a6b}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {c346e8a3-cbed-4d27-87cc-a70eb4dc8c27}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {ffc79924-5eaf-4666-a736-06190f281443}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {2364d06a-f67f-4186-aed0-62b99e1784f1}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {464bdb3c-91c4-4e9b-896f-225496ac4ed6}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {28566328-f041-4466-8b14-8f5831e78f8b}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {6b4a94db-54fe-4ae1-9be4-7a7dad004600}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10400:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {8c56eb1e-2b47-466f-8d33-7dbcd63f3df2}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {5b11d51b-2f4c-4452-bcc3-09f2a1160cc0}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {75fc75f7-c589-4a07-a25b-72e03b0383b3}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10400:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {107af0e0-ef1a-4d19-aba8-67a163073d13}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10400:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {07cfaffb-5a2e-4b99-b62a-e4ca53b6d5aa}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {463707f8-a1d0-4585-876d-83aa6d60b89e}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {a4c749ef-6ecf-48aa-8448-50a7a1165ff7}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10400:00:01 - FFmpeg:  10500:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {76988a52-df13-419a-8e64-ffcf4a336cf5}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {80a3a7bd-89d8-4497-a2b8-2126af7e6eb8}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10400:00:01 - FFmpeg:  10500:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {50925b7b-e931-4978-a12a-586630f095f9}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {b69c20e0-2508-8790-0305-875499e0a2d0}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] {49761bec-4b63-4349-a5ff-87ffdf088466}00:00:01 - FFmpeg:  10300:00:01 - FFmpeg:  10600:00:01 - FFmpeg: 
00:00:01 - FFmpeg: [hevc @ 00000287DFD13F80] Output frame with POC 3.
00:00:01 - SDL Info (0): FFmpeg-based video decoder chosen
00:00:02 - Qt Info: Found "gamecontrollerdb.txt" at "C:/Users/user/Downloads/MoonlightPortable-x64-r1685/cache/gamecontrollerdb.txt"
00:00:02 - SDL Info (0): Loaded 430 new gamepad mappings
00:00:02 - Qt Info: "FLOOR2" is now online at "192.168.0.4:47989"
00:00:02 - Qt Debug: Current Moonlight version: "3.2.0"
00:00:02 - Qt Info: Gamepad mappings are up to date
00:00:02 - Qt Info: Latest supported GFE server: "99.99.99.99"
00:00:02 - Qt Info: Discovered mDNS host: "floor2.local."
00:00:02 - Qt Info: Resolved "floor2.local." to QHostAddress("192.168.0.4")
00:00:02 - Qt Info: Resolved "floor2.local." to QHostAddress("fe80::3509:3c3:661:a3c5")
00:00:03 - Qt Debug: Found update manifest match for current platform
00:00:03 - Qt Debug: Latest version of Moonlight for this platform is: "3.2.0"
00:00:03 - Qt Debug: Update manifest version equal to current version
00:00:04 - Qt Info: Processing new PC at "192.168.0.4:47989" from mDNS with IPv6 address "<NULL>"
00:00:04 - Qt Info: Executing request: "http://192.168.0.4:47989/serverinfo?uniqueid=0123456789ABCDEF&uuid=f2a5738558744017bdc53773869ccd47"
00:00:04 - Qt Info: Executing request: "https://192.168.0.4:47984/serverinfo?uniqueid=0123456789ABCDEF&uuid=b927c830e6ff4423a463f20e412307c7"
00:00:14 - Qt Info: Found "ModeSeven.ttf" at ":/data/ModeSeven.ttf"
00:00:14 - Qt Info: Server GPU: "NVIDIA GeForce RTX 2070 SUPER"
00:00:14 - Qt Info: Server GFE version: "3.25.0.84"
00:00:14 - SDL Info (0): Video bitrate: 150000 kbps
00:00:14 - SDL Info (0): Desired audio buffer: 240 samples (960 bytes)
00:00:14 - SDL Info (0): Obtained audio buffer: 240 samples (960 bytes)
00:00:14 - SDL Info (0): Audio channel count: 2
00:00:14 - SDL Info (0): Audio channel mask: 3
00:00:14 - SDL Info (0): V-sync disabled
00:00:14 - SDL Info (0): Windowed mode with DWM running
00:00:14 - SDL Info (0): Windowed: 1 | Present Interval: 80000000
00:00:14 - SDL Info (0): Detected GPU: Intel(R) Iris(R) Plus Graphics 655 (8086:3ea5)
00:00:14 - SDL Info (0): GPU driver: igdumdim64.dll 27.20.100.8681
00:00:14 - SDL Info (0): Avoiding IDirectXVideoProcessor API on Intel GPU
00:00:14 - SDL Info (0): Using DXVA2 accelerated renderer
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 32(VPS), nuh_layer_id: 0, temporal_id: 0
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 33(SPS), nuh_layer_id: 0, temporal_id: 0
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 34(PPS), nuh_layer_id: 0, temporal_id: 0
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 21(CRA_NUT), nuh_layer_id: 0, temporal_id: 0
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding VPS
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Main profile bitstream
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding SPS
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Main profile bitstream
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding VUI
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding PPS
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Format dxva2_vld chosen by get_format().
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Format dxva2_vld requires hwaccel initialisation.
00:00:14 - SDL Info (0): DXVA2 decoder surface high-water mark: 0
00:00:14 - FFmpeg: [hevc @ 00000287DFFADF40] Output frame with POC 3.
00:00:14 - SDL Info (0): FFmpeg-based video decoder chosen
00:00:14 - SDL Info (0): Desired audio buffer: 240 samples (960 bytes)
00:00:14 - SDL Info (0): Obtained audio buffer: 240 samples (960 bytes)
00:00:15 - Qt Info: Found "gamecontrollerdb.txt" at "C:/Users/user/Downloads/MoonlightPortable-x64-r1685/cache/gamecontrollerdb.txt"
00:00:15 - SDL Info (0): Loaded 430 new gamepad mappings
00:00:15 - SDL Info (0): V-sync disabled
00:00:15 - SDL Info (0): Windowed mode with DWM running
00:00:15 - SDL Info (0): Windowed: 1 | Present Interval: 80000000
00:00:15 - SDL Info (0): Detected GPU: Intel(R) Iris(R) Plus Graphics 655 (8086:3ea5)
00:00:15 - SDL Info (0): GPU driver: igdumdim64.dll 27.20.100.8681
00:00:15 - SDL Info (0): Avoiding IDirectXVideoProcessor API on Intel GPU
00:00:15 - SDL Info (0): Using DXVA2 accelerated renderer
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] nal_unit_type: 32(VPS), nuh_layer_id: 0, temporal_id: 0
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] nal_unit_type: 33(SPS), nuh_layer_id: 0, temporal_id: 0
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] nal_unit_type: 34(PPS), nuh_layer_id: 0, temporal_id: 0
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] nal_unit_type: 21(CRA_NUT), nuh_layer_id: 0, temporal_id: 0
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Decoding VPS
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Main profile bitstream
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Decoding SPS
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Main profile bitstream
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Decoding VUI
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Decoding PPS
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Format dxva2_vld chosen by get_format().
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Format dxva2_vld requires hwaccel initialisation.
00:00:15 - SDL Info (0): DXVA2 decoder surface high-water mark: 0
00:00:15 - FFmpeg: [hevc @ 00000287DFFAE340] Output frame with POC 3.
00:00:15 - SDL Info (0): FFmpeg-based video decoder chosen
00:00:15 - SDL Info (0): Avoiding IDirectXVideoProcessor API on Intel GPU
00:00:15 - Qt Info: Found "gamecontrollerdb.txt" at "C:/Users/user/Downloads/MoonlightPortable-x64-r1685/cache/gamecontrollerdb.txt"
00:00:15 - SDL Info (0): Loaded 430 new gamepad mappings
00:00:16 - Qt Info: Executing request: "https://192.168.0.4:47984/resume?uniqueid=0123456789ABCDEF&uuid=9b70673b0f014a1bb46babde137d4422&rikey=bbb7446896de54fd17d6168dcf2a6ba4&rikeyid=569476782&surroundAudioInfo=196610"
00:00:17 - Qt Info: Resume response: "<?xml version=\"1.0\" encoding=\"UTF-16\"?><root protocol_version=\"0.1\" query=\"resume\" status_code=\"200\" status_message=\"OK\"><DisplayHeight>720</DisplayHeight><DisplayWidth>1280</DisplayWidth><HdrMode>0</HdrMode><RefreshRate>60</RefreshRate><avkey>ok</avkey><gamesession>ca0ad29c286d164f50599629c19a872e</gamesession><resume>1</resume><rikey>ok</rikey><sessionUrl0>rtsp://192.168.0.4:48010</sessionUrl0></root>"
00:00:17 - Qt Info: Found matching interface: "Ethernet" "94:C6:91:A7:18:44" QFlags<QNetworkInterface::InterfaceFlag>(IsUp|IsRunning|CanBroadcast|CanMulticast)
00:00:17 - Qt Info: Interface Type: QNetworkInterface::Ethernet
00:00:17 - Qt Info: Interface MTU: 1500
00:00:17 - SDL Info (0): RTSP port: 48010
00:00:17 - SDL Info (0): Initializing platform...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Resolving host name...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Initializing audio stream...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Starting RTSP handshake...
00:00:17 - SDL Info (0): Audio port: 48000
00:00:17 - SDL Info (0): Video port: 47998
00:00:17 - SDL Info (0): Control port: 47999
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Initializing control stream...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Initializing video stream...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Initializing input stream...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Starting control stream...
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Starting video stream...
00:00:17 - SDL Info (0): Video stream is 2560x1440x60 (format 0x100)
00:00:17 - SDL Info (0): done
00:00:17 - SDL Info (0): Starting audio stream...
00:00:18 - SDL Info (0): Received first video packet after 0 ms
00:00:18 - SDL Info (0): Desired audio buffer: 240 samples (960 bytes)
00:00:18 - SDL Info (0): Obtained audio buffer: 240 samples (960 bytes)
00:00:18 - SDL Info (0): Audio stream has 2 channels
00:00:18 - SDL Info (0): done
00:00:18 - SDL Info (0): Starting input stream...
00:00:18 - SDL Info (0): done
00:00:18 - SDL Info (0): Received first audio packet after 0 ms
00:00:18 - SDL Info (0): Initial audio resync period: 625 milliseconds
00:00:18 - SDL Info (0): Qt UI screen is at (0,0)
00:00:18 - SDL Info (0): SDL found matching display 0
00:00:18 - SDL Info (0): Video decode unit queue overflow
00:00:18 - SDL Info (0): IDR frame request sent
00:00:18 - SDL Info (0): Video decode unit queue overflow
00:00:18 - SDL Info (0): IDR frame request sent
00:00:18 - SDL Info (0): Waiting for IDR frame
00:00:18 - SDL Info (0): Video decode unit queue overflow
00:00:18 - SDL Info (0): IDR frame request sent
00:00:18 - SDL Info (0): Waiting for IDR frame
00:00:18 - SDL Info (0): Recreating renderer for window event: 1 (0 0)
00:00:18 - SDL Info (0): Chosen best display mode: 3840x2160x60
00:00:18 - SDL Info (0): V-sync disabled
00:00:18 - SDL Info (0): Video decode unit queue overflow
00:00:18 - SDL Info (0): IDR frame request sent
00:00:18 - SDL Info (0): V-Sync disabled in tearing mode
00:00:18 - SDL Info (0): Windowed: 0 | Present Interval: 80000000
00:00:18 - SDL Info (0): Detected GPU: Intel(R) Iris(R) Plus Graphics 655 (8086:3ea5)
00:00:18 - SDL Info (0): GPU driver: igdumdim64.dll 27.20.100.8681
00:00:18 - SDL Info (0): Avoiding IDirectXVideoProcessor API on Intel GPU
00:00:18 - SDL Info (0): Frame pacing disabled: target 60 Hz with 60 FPS stream
00:00:18 - SDL Info (0): Using DXVA2 accelerated renderer
00:00:18 - SDL Info (0): FFmpeg-based video decoder chosen
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 32(VPS), nuh_layer_id: 0, temporal_id: 0
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 33(SPS), nuh_layer_id: 0, temporal_id: 0
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 34(PPS), nuh_layer_id: 0, temporal_id: 0
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] nal_unit_type: 19(IDR_W_RADL), nuh_layer_id: 0, temporal_id: 0
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding VPS
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Main profile bitstream
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding SPS
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Main profile bitstream
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding VUI
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Decoding PPS
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Format dxva2_vld chosen by get_format().
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Format dxva2_vld requires hwaccel initialisation.
00:00:18 - SDL Info (0): DXVA2 decoder surface high-water mark: 0
00:00:18 - FFmpeg: [hevc @ 00000287DFFADF40] Output frame with POC 0.
00:00:18 - SDL Info (0): Dropping window event during flush: 6 (3840 2160)
00:00:18 - SDL Info (0): Requesting IDR frame on behalf of DR
00:00:18 - SDL Info (0): IDR frame request sent
00:00:18 - SDL Info (0): Waiting for IDR frame
00:00:18 - SDL Info (0): DXVA2 decoder surface high-water mark: 1
00:00:18 - SDL Info (0): DXVA2 decoder surface high-water mark: 2
00:00:36 - SDL Info (0): Detected quit key combo
00:00:36 - SDL Info (0): Quit event received
00:00:36 - SDL Info (0): Raising 3 keys
00:00:36 - SDL Info (0): Video decode unit queue overflow
00:00:36 - SDL Info (0): IDR frame request sent
00:00:36 - SDL Info (0): Global video stats
00:00:36 - SDL Info (0): ----------------------------------------------------------
Incoming frame rate from network: 60.71 FPS
Decoding frame rate: 60.71 FPS
Rendering frame rate: 60.71 FPS
Frames dropped by your network connection: 0.58%
Frames dropped due to network jitter: 0.00%
Average network latency: 1 ms (variance: 0 ms)
Average decoding time: 2.48 ms
Average frame queue delay: 0.05 ms
Average rendering time (including monitor V-sync latency): 0.30 ms
00:00:36 - SDL Info (0): Video decode unit queue overflow
00:00:36 - SDL Info (0): IDR frame request sent
00:00:36 - SDL Info (0): Waiting for IDR frame
00:00:36 - SDL Info (0): Unable to recover audio data block 3840 to 3843 (2+0=2 received < 4 needed)
00:00:36 - SDL Info (0): Unrecoverable frame 1150: 18+0=18 received < 86 needed
00:00:36 - SDL Info (0): Network dropped 2 frames (frames 1150 to 1151)
00:00:36 - SDL Info (0): Waiting for IDR frame
00:00:36 - SDL Info (0): IDR frame request sent
00:00:36 - SDL Info (0): Waiting for IDR frame
00:00:36 - SDL Info (0): Stopping input stream...
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Stopping audio stream...
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Stopping video stream...
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Stopping control stream...
00:00:36 - SDL Info (0): ENet peer acknowledged disconnection
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Cleaning up input stream...
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Cleaning up video stream...
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Cleaning up control stream...
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Cleaning up audio stream...
00:00:36 - Qt Info: Found "gamecontrollerdb.txt" at "C:/Users/user/Downloads/MoonlightPortable-x64-r1685/cache/gamecontrollerdb.txt"
00:00:36 - SDL Info (0): Loaded 430 new gamepad mappings
00:00:36 - SDL Info (0): done
00:00:36 - SDL Info (0): Cleaning up platform...
00:00:36 - SDL Info (0): done
00:00:37 - Qt Info: Discovered mDNS host: "floor2.local."
00:00:37 - Qt Info: Resolved "floor2.local." to QHostAddress("192.168.0.4")
00:00:37 - Qt Info: Resolved "floor2.local." to QHostAddress("fe80::3509:3c3:661:a3c5")
00:00:37 - Qt Info: Resolved "floor2.local." to QHostAddress("192.168.0.4")
00:00:37 - Qt Info: Resolved "floor2.local." to QHostAddress("fe80::3509:3c3:661:a3c5")
00:00:39 - Qt Info: Processing new PC at "192.168.0.4:47989" from mDNS with IPv6 address "<NULL>"
00:00:39 - Qt Info: Executing request: "http://192.168.0.4:47989/serverinfo?uniqueid=0123456789ABCDEF&uuid=efe2ea5363ed4b4fb8dc7e9b75b031ce"
00:00:39 - Qt Info: Executing request: "https://192.168.0.4:47984/serverinfo?uniqueid=0123456789ABCDEF&uuid=a53399fec76c4115be0c0d2cd66af765"
```

**Comment by cgutman** [comment]
Can you describe your client PC's audio setup in detail?

**Comment by ottelo9** [comment]
I have audio problems too. My client PC is a Intel NUC mini computer connected to a LCD with HDMI audio out. I can’t hear any audio from my Host PC. But if I connect a headphone to my client and set the HDMI Out to default again I can hear sound.

**Comment by iTaybb** [comment]
My setup is identical to @ottelo9, Intel Nuc (8i3bek) connected to my TV through HDMI audio..
I can also confirm that connecting a headphone to the client works - I can hear through the headphones. But not through the HDMI out.

**Comment by cgutman** [comment]
Interesting, I have a NUC11PAHi3 connected via HDMI to my TV and didn't have any audio issues.

Please try the following to narrow down the possible cause:

First, try windowed or borderless windowed in Moonlight to see if the full-screen mode change is causing the audio issue.

Second, open a Command Prompt, run:
```
set ML_AUDIO=libsoundio
"C:\Program Files\Moonlight Game Streaming\Moonlight.exe"
```

If you're running Moonlight Portable, you'll need to change that path in the second line.

**Comment by iTaybb** [comment]
Both methods seem to work fine.
Borderless and windowed methods are working perfectly with audio.
Also using the full screen method with setting `ML_AUDIO=libsoundio` gets the audio to work again.

**Comment by ottelo9** [comment]
> set ML_AUDIO=libsoundio

Works fine for me :)
Can anyone put this info inside the FAQ?

**Comment by cgutman** [comment]
Can you attach your Moonlight log file after encountering the bug?

For portable builds, it will be in the directory with Moonlight.exe. For installed builds, it will be in `%TEMP%`.

**Comment by ottelo9** [comment]
[Moonlight-1645100073.log](https://github.com/moonlight-stream/moonlight-qt/files/8089133/Moonlight-1645100073.log)

**Comment by teconmoon** [comment]
I'm also experiencing this running moonlight on my Intel N4000 based mini PC. Running in Borderless Windowed mode works around the issue, as does using headphones instead of HDMI audio.

I punched the environmental variable that was recommended above and that worked for me. I can also collect logs if it helps correlate a permanent solution.

Would you like logs with or without the environmental variable applied?

**Comment by cgutman** [comment]
I was able to reproduce this issue by hotplugging the display while streaming and implemented a fix.

Until the next version of Moonlight is released, you can use a nightly build: https://ci.appveyor.com/project/cgutman/moonlight-qt/builds/42800238/job/s96nxqtymr4lrabb/artifacts

**Comment by ottelo9** [comment]
Its now included in v4.0.0
